status-loop/queryValidators.js

```diff
@@ -52,6 +52,28 @@ function getStatus (messagesFromAll, campaign, balanceTree) {
 	throw new Error('internal error: no status detected; should never happen')
 }
 
+function getHumanFriendlyName (status, campaign) {
+	if (campaign.status && campaign.status.humanFriendlyName === 'Closed') return 'Closed'
+	switch (status) {
+	case 'Active':
```

Comment:
shouldn't this be idented? other than that, LGTM

Reply:
based on the eslint configuration there shouldn't be any indentation here.
I am always formatting with the eslint config on save. I suggest we copy the eslint config from the platform. What do you think?

Reply:
ok - leave it as is, then in a separate PR, copy the cfg from the platform and format everything